include/aos/common/alerts/sender.hpp

```diff
@@ -4,41 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef AOS_ALERTS_HPP_
-#define AOS_ALERTS_HPP_
+#ifndef AOS_ALERTS_SENDER_HPP_
+#define AOS_ALERTS_SENDER_HPP_
 
 #include "aos/common/cloudprotocol/alerts.hpp"
 #include "aos/common/types.hpp"
 
 namespace aos::alerts {
 
-/**
```

Comment:
Let's keep this interface in `alerts.hpp` to be aligned with other modules.

Reply:
done